docs/manage/CLI.md

```diff
@@ -9,12 +9,13 @@ aliases:
     - "/program/cli"
```

Comment:
Can we move the CLI instructions to be under "Installation Guide" ?

Reply:
The CLI is not used for installing Viam (anything) other than the CLI itself. I understand how you and eng might find this confusing: let's consider alternate IA (left-nav, naming) after we get all the facts correct and the salient procedurals down.